libr/core/cmd_type.c

```diff
@@ -375,6 +448,23 @@ static int stdifstruct(void *user, const char *k, const char *v) {
 	return false;
 }
 
+static int print_struct_union_list_json_cb(void *p, const char *k, const char *v) {
+	RCore *core = (RCore *)p;
+	PJ *pj = pj_new ();
+	pj_o (pj);
+	Sdb *sdb = core->anal->sdb_types;
+	char *sizecmd = r_str_newf ("%s.%s.size", v, k);
+	char *size_s = sdb_querys (sdb, NULL, -1, sizecmd);
+	pj_ks (pj, "type", k);
+	pj_ki (pj, "size", size_s ? atoi (size_s) : 0);
+	pj_end (pj);
+	r_cons_printf ("%s", pj_string (pj));
```

Comment:
use "%s\n"

Reply:
This would result in newlines between json objects:
```
[{"type":"a","size":1400}
,{"type":"addr","size":1368}
]
```
rather than this:
`[{"type":"a","size":1400}, {"type":"addr","size":1368}]`
Should I still make the changes?

Reply:
json object should be executed only at the end. Also, instead of "\n" use `r_cons_println (pj_string (pj));`